Weigh search results by age

simenheg opened the pull request:
Weigh search results by their age by default. Add a new setting, `SEARCH_AGE_SCALE_FACTOR`, controlling how much emphasis to put on the age when ranking results (set this to 0 to revert to the old behavior).

stephenmcd commented:
Thanks a lot for this, I think it's a great idea! I think the implementation could be better though:

- No need for a setting to disable it, just have it always on 
- No need for the "6 month" drop off, have a look at the time scaling algorithm here: https://github.com/stephenmcd/drum/blob/c92eff9952b340256e1c02ee92151630ecbf2986/drum/links/utils.py#L48-L50

simenheg commented:
Thanks your feedback, @stephenmcd. I've updated the commit addressing your comments.

I did adjust the default the default scaling factor from 2 to 1.5 because a factor of 2 seemed to bring too many irrelevant results to the first page because of their freshness in my testing, while 1.5 seemed to provide a good balance.

stephenmcd commented:
Really awesome, thanks again!
